app/models/user.rb

```diff
@@ -9,7 +10,7 @@ class User < ApplicationRecord
   validates :uid, presence: true
   validates :pbs_id, presence: true, allow_blank: true
 
-  enum role: %i[user admin programm], _prefix: :role
+  bitfield :role_flags, :role_user, :role_admin, :role_programm
```

Comment:
this allows us multiple roles, right? is it easily adaptable if we would add more roles?

Reply:
Just add more here, should work fine